frontend/src/components/Users/UserInfo/UserInfo.test.js

```diff
@@ -162,6 +162,108 @@ describe("UserInfo", () => {
         expect(await screen.findByText("INACTIVE")).toBeInTheDocument();
         expect(await screen.findByText("LOCKED")).toBeInTheDocument();
     });
+
+    test("update the division", async () => {
+        const user = {
+            full_name: "Test User",
+            email: "test.user@exampl.com",
+            division: 1,
+            status: "ACTIVE",
+            roles: ["admin"]
+        };
+        const { getByText, container } = renderWithProviders(
+            <App
+                user={user}
+                isEditable={true}
+            />
+        );
+
+        await waitFor(() => {
+            expect(container).toBeInTheDocument();
+        });
+
+        expect(await screen.findByText("User Details")).toBeInTheDocument(); // Card Header
+
+        // find the input element within the div with testid division-combobox
+        const divisionComboBox = screen.getByTestId("division-combobox");
+        // eslint-disable-next-line testing-library/no-node-access
+        const divisionInput = divisionComboBox.querySelector("input");
+
+        fireEvent.keyDown(divisionInput, { key: "ArrowDown", code: 40 });
```

Comment:
learned that the `[userEvent](https://testing-library.com/docs/user-event/intro/)` is now the preferred method of handling user interactions